crates/ide/src/rename.rs

```diff
@@ -27,6 +27,23 @@ pub use ide_db::rename::RenameError;
 
 type RenameResult<T> = Result<T, RenameError>;
 
+pub struct RenameConfig {
+    pub prefer_no_std: bool,
+    pub prefer_prelude: bool,
+    pub prefer_absolute: bool,
+}
+
+impl RenameConfig {
+    fn import_path_config(&self) -> ImportPathConfig {
+        ImportPathConfig {
+            prefer_no_std: self.prefer_no_std,
+            prefer_prelude: self.prefer_prelude,
+            prefer_absolute: self.prefer_absolute,
+            allow_unstable: true,
+        }
+    }
```

Comment:
Should rename config maybe just contain an `ImportPathConfig`? (and overwrite the `allow_unstable` field on construction)